tests/integration/ci-runner.py

```diff
@@ -353,6 +353,11 @@ def _install_clickhouse(self, debs_path):
         )
 
     def _compress_logs(self, dir, relpaths, result_path):
+        # We execute sync in advance to have all files written after containers
+        # are finished or killed
+        subprocess.check_call(  # STYLE_CHECK_ALLOW_SUBPROCESS_CHECK_CALL
+            "sync", shell=True
+        )
```

Comment:
This looks odd, `syncfs` is to sync to the underlying backing storage, it should not be required to read those files. The problem with `file changed as we read it` is because the process was still working while integration test runner finished.

I've seen this couple of times before, but anyway now there is #44529, and this should not be required anymore.

Reply:
yes, we can revert it